provider.swagger.yml

```diff
@@ -114,8 +121,14 @@ definitions:
     required:
       - message
     properties:
+      code:
+        type: integer
+        required: false
+        description: >
+          An agreed upon error code. example: 100 (slug conflict.)
```

Comment:
Vous avez tous les deux raison, et en fait cette spécification ne force pas de code d'erreur. Tout code d'erreur http est couvert par le statut "default" [dans la spec d'OpenAPI](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#responses-object) : *"The default can be used as the default response object for all HTTP codes that are not covered individually by the specification."*

Je suis donc plutôt pour : 

 * laisser chaque *provider* employer les codes http d'erreur comme ils l'entend, en encourageant simplement à avoir un usage pertinent d'HTTP ;
 * ajouter des codes d'erreur à la discrétion de chaque *provider*, sans chercher à normer tous les cas.

Le 2e point n'est pas idéal - ce serait mieux que ces codes soient normés, mais je pense que nous n'avons pas le temps de réfléchir aux différents cas possibles et au format de code qui serait le plus efficace. Je propose donc de remettre cette spécification précise des codes d'erreur à plus tard.